src/main/java/frc/robot/hardware/request/cansparkmax/SparkMaxRequestBuilder.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.hardware.request.cansparkmax;
+
+import com.revrobotics.CANSparkBase;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+import java.util.function.Function;
+
+public class SparkMaxRequestBuilder {
+
+	//@formatter:off
```

Comment:
What does the formatter do? If you manually need to turn it on and off I consider that a bug in the formatter config.

Reply:
The func name line down, and then one line from then

Reply:
This can be fixed by changing `/codestyle.xml:124`
```diff
-        <setting id="org.eclipse.jdt.core.formatter.alignment_for_method_declaration" value="16"/>
+        <setting id="org.eclipse.jdt.core.formatter.alignment_for_method_declaration" value="0"/>
```
Wanna sneak that into this PR or should I open a separate one?

Reply:
another pr
there are a lot of places of that in code need to remove format off from them

Reply:
on it